test-infra/camel-test-infra-core/src/test/java/org/apache/camel/test/infra/core/impl/CamelTestSupport.java

```diff
@@ -191,14 +192,18 @@ public void setCamelContextService(Service service) {
     // MockEndpoint.resolve(context, uri);
 
 
-    @Override
-    public void configureContext(CamelContext context) throws Exception {
+    protected abstract void configureCamelContext(CamelContext context);
 
 
+    @Override
+    @ContextFixture
```

Comment:
It was missing the `@ContextFixture` annotation, so the extension was never calling the context configuration method.